src/Symfony/Component/Cache/Adapter/PdoAdapter.php

```diff
@@ -38,9 +38,9 @@ class PdoAdapter extends AbstractAdapter implements PruneableInterface
      *
      * @param \PDO|Connection|string $connOrDsn a \PDO or Connection instance or DSN string or null
      *
-     * @throws InvalidArgumentException When first argument is not PDO nor Connection nor string
-     * @throws InvalidArgumentException When PDO error mode is not PDO::ERRMODE_EXCEPTION
-     * @throws InvalidArgumentException When namespace contains invalid characters
+     * @throws \InvalidArgumentException When first argument is not PDO nor Connection nor string
+     * @throws \InvalidArgumentException When PDO error mode is not PDO::ERRMODE_EXCEPTION
+     * @throws \InvalidArgumentException When namespace contains invalid characters
```

Comment:
should be reverter: what's missing is a `use Symfony\Component\Cache\Exception\InvalidArgumentException;`

Reply:
Thanks! I changed it to the correct Class